### PR TITLE
Updated the Instructions for use of the label bot

### PR DESCRIPTION
**Pull request by harshp8l**
Within CONTRIBUTORS.md file, instructions for use of the label bot have been updated to include the template for the commands it can perform. These commands are: add, remove, and update.

### CODE REVIEW

**Comment by harshp8l** [comment]
@mxnet-label-bot add [pr-awaiting-review]

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -190,10 +190,14 @@ List of Contributors
...
 * [Rahul Padmanabhan](https://github.com/rahul3)
+* [Harsh Patel](https://github.com/harshp8l)
 
 Label Bot
 ---------
 * [mxnet-label-bot](https://github.com/mxnet-label-bot)
   - mxnet-label-bot provides users with the functionality to add labels from the repository to issues
-  - To use me, comment: @mxnet-label-bot add [specify label here]
+  - To use me, comment: <br>
+  @mxnet-label-bot add [specify labels here] <br>
```
Comment (nswamy) [inline CONTRIBUTORS.md]: [specify comma separated labels]?

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -191,10 +191,15 @@ List of Contributors
...
 Label Bot
 ---------
 * [mxnet-label-bot](https://github.com/mxnet-label-bot)
-  - mxnet-label-bot provides users with the functionality to add labels from the repository to issues
-  - To use me, comment: @mxnet-label-bot add [specify label here]
+  - mxnet-label-bot provides users with the functionality to manage labels for Issues/Pull Requests on the repository
+  - To use me, comment:
+    - @mxnet-label-bot add [specify comma separated labels here]
+    - @mxnet-label-bot remove [specify comma separated labels here]
+    - @mxnet-label-bot update [specify comma separated labels here] <br>
```
Comment (sandeep-krishnamurthy) [inline CONTRIBUTORS.md]: why <br>?
You can get a new line without bullet, by using 4 space I guess.

Comment (sandeep-krishnamurthy) [reply CONTRIBUTORS.md]: Can you add this link for a list of available label names - https://github.com/apache/incubator-mxnet/labels

**Comment by kalyc** [comment]
@mxnet-label-bot update [pr-awaiting-merge]

**Pull request merged**